src/ibek/support_cmds/checks.py

```diff
@@ -82,6 +83,37 @@ def do_dependencies():
     RELEASE_SH.write_text(shell_text)
 
 
+def check_deps(deps: list[str]) -> None:
+    """
+    Check if specified dependencies have been supplied
```

Comment:
spelling 'dependencies'

I use extension streetsidesoftware.code-spell-checker in vscode. Its pretty good at not getting too many false positives. If you like grammarly you might like this.